### DISCUSSION TITLE
Extraction of MS/MS spectra from different collision energy acquired by Agilent 6545 TOF

### DISCUSSION THREAD

**Discussion by QizhiSu** [systemsomicslab/MsdialWorkbench#191] [Q&A]
Hi, 

We are currently using Agilent 6545 TOF. Since there are 3 collision energy applied, normally 10, 20, 40V, I am curious which MS/MS spectra will be used by MS-DIAL? Is there any way to select the spectra that I want or even average spectra from all the 3 collision energies?

Thanks in advance.

Sukis

**Comment by kozo2** [comment]
Hi @QizhiSu ,
Thank you for the question. I believe this is a pure question unrelated to the code, so I will convert it from a GitHub Issue to a GitHub Discussion Q&A.

**Comment by YukiMatsuzawa** [comment]
As of the current version of MSDIAL, there isn't a feature to display average spectra for Collision energy. However, you can display other raw MS/MS data as shown in the attached figure.

<img width="955" alt="image" src="https://github.com/systemsomicslab/MsdialWorkbench/assets/122433968/4dbc01b4-1daa-46b6-8698-fa17659fccd7">

**Comment by QizhiSu** [comment]
Hi YukiMatsuzawa,

Thanks a lot for your prompt response. It is good to know the way to see MS/MS spectra under different collision energy.
Still, I am wondering which spectrum will be selected during the alignment process?

All the best, 
Sukis
